python/ray/tune/experiment.py

```diff
@@ -155,8 +155,7 @@ def __init__(self,
             "checkpoint_score_attr": checkpoint_score_attr,
             "export_formats": export_formats or [],
             "max_failures": max_failures,
-            "restore": os.path.abspath(os.path.expanduser(restore))
```

Comment:
Can you move this code to `create_trial_from_spec`?

Reply:
Like this? https://github.com/thavlik/ray/commit/fb8011a702807d9dd9c6636c60caf235a05ee7d2

Reply:
Like this? https://github.com/thavlik/ray/commit/fb8011a702807d9dd9c6636c60caf235a05ee7d2 (also, I just fixed the typo there - whoops)

Reply:
Looks good